### PR TITLE
[353] Add Liberty profile for build/deploy

### LINKED ISSUE #353: Add Liberty profile to allow easy deployment/testing in Open Liberty
I've been running into some issues building the MP Starter app - I think the errors are related to deployment in Thorntail - basically the tests are failing.  In one case, they fail to compile in the default profile, but they also fail at runtime using the `-Pthorntail` profile.  This might just be my general lack of understanding with Thorntail.

In order to quickly and easily test my changes, I ended up creating a Liberty profile enabling me to quickly build and deploy the mp-starter.war into an Open Liberty server.  With this change I could build, deploy and run the app with: `mvn package -Pliberty liberty:run`. Then I could test it (manually) by browsing to: http://localhost:9080/mp-starter/

I'd like to share this so that others might be able to build and test similarly.

### PR DESCRIPTION
**Pull request by andymc12**
Adds liberty maven profile to quickly build and deploy to Open Liberty server for manual testing. Closes #353.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -30,4 +30,6 @@ Information about adding
 * an additional MicroProfile Version for an Implementation
 * a new MicroProfile Specification
 
-is found in the [how to document](https://github.com/eclipse/microprofile-starter/blob/master/how-to.md).
\ No newline at end of file
+is found in the [how to document](https://github.com/eclipse/microprofile-starter/blob/master/how-to.md).
+
+It is also possible to build and deploy to Open Liberty using the Liberty profile. To build the MP Starter app, use: `mvn package -Pliberty liberty:run` and then browse to http://localhost:9080/mp-starter/ to test the app.
```
Comment (Emily-Jiang) [inline README.md:35]: You should add the above info [here](https://github.com/eclipse/microprofile-starter/tree/master/Container)

Comment (andymc12) [reply README.md]: Thanks for the suggestion @Emily-Jiang - I have moved it there.

**Review by Emily-Jiang — APPROVED** [review]
LGTM! Thanks @andymc12 !

**Comment by Emily-Jiang** [comment]
@rdebusscher @Karm do you have comments for this?

**Review by rdebusscher — APPROVED** [review]
LGTM

**Pull request merged**